WarehouseKing.java

```diff
@@ -0,0 +1,22 @@
+import Java.util.ArrayList; 
+
+public class WarehouseKing{
+	
+	private ArrayList <Store> StoreBranches;
+
+	public String storeFinder(String productName){
```

Comment:
What does this method do? Just searches stores to returns a string of which stores has the products? If that's the intention, why not return a list of store objects, so we can continue manipulating them later? :)